application/main.py

```diff
@@ -315,7 +315,7 @@ def process_upload_validation(files, validation_config, user_id, commit_id=None,
         user = session.query(database.user).filter(database.user.id == user_id).all()[0]
 
     msg = f"{len(filenames)} file(s) were uploaded by user {user.name} ({user.email}): {(', ').join(filenames)}"
-    utils.send_message(msg, os.getenv("DEV_EMAIL"))
+    utils.send_message(msg, [os.getenv("CONTACT_EMAIL")])
```

Comment:
All looks good, just one small comment.

When you don't supply these env vars my guess is the program will error, but only very late in the process.

Somewhere near the top of main.py I would do a couple of `assert os.getenv("CONTACT_EMAIL")` `assert ....` so that the server wouldn't start even if the vars are not present.

Or alternatively,

```py
if os.getenv("CONTACT_EMAIL"):
    utils.send_message(msg, [os.getenv("CONTACT_EMAIL")])
else:
    logger.warn("No CONTACT_EMAIL set, not sending upload notification")
```